### PR TITLE
Updates to README file

### PR DESCRIPTION
**Pull request by sboisvert**
Add a missing NPM step
Clarify where to get .env variables
Add French translation
Remove extra URL tags to be compatible with pipenv
See [pypa/pipenv#2057 (comment)](https://github.com/pypa/pipenv/issues/2057#issuecomment-384594549)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,11 +86,7 @@ source  ~/.pyenv/versions/3.6.9/bin/virtualenvwrapper.sh
 
 `workon notifications-admin`
 
-9. Decrypt our existing set of environment variables
-
-`aws kms decrypt --ciphertext-blob fileb://.env.enc.aws --output text --query Plaintext --region us-east-1 | base64 --decode > .env`
-
-A sane set of defaults exists in `.env.example`
+9. Find the appropriate env variables and copy them into the .env file. A sane set of defaults exists in `.env.example` in the root folder or you can use the ones in the LastPass folder. If using from lastPass and running the API locally, change API_HOST_NAME to point to your local machine
```
Comment (brdunfield) [inline README.md:89]: For CDS Notify Devs - LastPass should always be used. We can maybe make this paragraph a little more clear given that?

File: requirements-app.txt [hunk 2]
```diff
@@ -36,6 +36,6 @@ awscli-cwlogs>=1.4,<1.5
 # Putting upgrade on hold due to v1.0.0 using sha512 instead of sha1 by default
 itsdangerous==0.24  # pyup: <1.0.0
 
-git+https://github.com/cds-snc/notifier-utils.git@43.0.1#egg=notifications-utils==43.0.1
+git+https://github.com/cds-snc/notifier-utils.git@43.0.1#egg=notifications-utils
```
Comment (brdunfield) [inline requirements-app.txt:39]: What does this change do?

Comment (brdunfield) [reply requirements-app.txt:39]: Oh this is the pipenv change

**Review by brdunfield — CHANGES_REQUESTED** [review]
I think we can make the .env variable paragraph a little more clear, and there is currently a merge conflict. Otherwise, looks great, and super happy to see the french translation! Thank you!

File: README.md [hunk 3]
```diff
@@ -207,3 +207,214 @@ See the [notification-api](https://github.com/cds-snc/notification-api) README f
...
 Note: Tests may break if `USE_LOCAL_JINJA_TEMPLATES` is set to `True` in your .env
+
+
+
+
+# Notifications-admin
+
+Application d'administration des notifications.
+
+## Branche amont (Upsteam)
```
Comment (AntoineAugusti) [inline README.md:218]:
```suggestion
## Branche amont (_upstream_)
```

File: README.md [hunk 4]
```diff
@@ -207,3 +207,214 @@ See the [notification-api](https://github.com/cds-snc/notification-api) README f
...
+
+Ce dépôt Git est une version modifiée de :
+https://github.com/alphagov/notifications-admin
+
+## Caractéristiques de cette application
+
+ - Enregistrer et gérer les utilisateurs
+ - Créer et gérer des services
+ - Envoyer des courriels et des SMS par lots en téléchargeant un CSV
+ - Afficher l'historique des notifications
```
Comment (AntoineAugusti) [inline README.md:228]:
```suggestion
- Enregistrer et gérer les utilisateurs
- Créer et gérer des services
- Envoyer des courriels et des SMS par lots en téléversant un CSV
- Afficher l'historique des notifications
```

File: README.md [hunk 5]
```diff
@@ -207,3 +207,214 @@ See the [notification-api](https://github.com/cds-snc/notification-api) README f
...
+
+- Nous ne pouvons pas actuellement envoyer des lettres
+- Nous ne pouvons pas savoir si les SMS ont été délivrés ou non
+
+## Première mise en place
+
+Brew est un gestionnaire de paquets pour OSX. La commande suivante permet d'installer brew :
+``shell
+    /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+```
```
Comment (AntoineAugusti) [inline README.md:240]:
```suggestion
```shell
/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
```
```

File: README.md [hunk 6]
```diff
@@ -207,3 +207,214 @@ See the [notification-api](https://github.com/cds-snc/notification-api) README f
...
+
+Langues nécessaires
```
Comment (AntoineAugusti) [inline README.md:242]:
```suggestion
Langages nécessaires
```

File: README.md [hunk 7]
```diff
@@ -207,3 +207,214 @@ See the [notification-api](https://github.com/cds-snc/notification-api) README f
...
+- Python 3.X
+- [Node](https://nodejs.org/) 10.15.3 ou supérieur
+- [npm](https://www.npmjs.com/) 6.4.1 ou plus
+"shell
+    brew install node
+```
```
Comment (AntoineAugusti) [inline README.md:248]:
```suggestion
```shell
brew install node
```
```

File: README.md [hunk 8]
```diff
@@ -207,3 +207,214 @@ See the [notification-api](https://github.com/cds-snc/notification-api) README f
...
+
+
+[NPM](npmjs.org) est l'outil de gestion des paquets de Node. `n` est un outil de gestion des
+différentes versions de Node. Ce qui suit installe `n` et utilise le support à long terme (LTS)
+version de Node.
+"shell
+    npm install -g n
+    n lts
+    npm rebuild node-sass
+```
```
Comment (AntoineAugusti) [inline README.md:258]:
```suggestion
[NPM](npmjs.org) est l'outil de gestion des paquets de Node. `n` est un outil de gestion des
différentes versions de Node. Ce qui suit installe `n` et utilise le support à long terme (LTS)
version de Node.
```shell
npm install -g n
n lts
npm rebuild node-sass
```
```

File: README.md [hunk 9]
```diff
@@ -207,3 +207,214 @@ See the [notification-api](https://github.com/cds-snc/notification-api) README f
...
+
+### Instruction d'installation locale 
+
+Sur OS X :
```
Comment (AntoineAugusti) [inline README.md:262]:
```suggestion
Sur macOS :
```

File: README.md [hunk 10]
```diff
@@ -207,3 +207,214 @@ See the [notification-api](https://github.com/cds-snc/notification-api) README f
...
+
+7. Redémarrez votre terminal et créez votre environnement virtuel :
+
+`mkvirtualenv -p ~/.pyenv/versions/3.6.9/bin/python notifications-admin`
+
+8. Vous pouvez maintenant retourner dans votre environnement à tout moment en entrant
+
+`workon notifications-admin`
+
+9. Trouvez les variables env appropriées et copiez-les dans le fichier .env. Un ensemble de valeurs par défaut existe dans le fichier `.env.example` à la racine ou vous pouvez utiliser celles du dossier LastPass. Si vous utilisez celles de LastPass et que vous exécutez l'API localement, modifiez API_HOST_NAME pour qu'elle pointe vers votre machine locale
```
Comment (AntoineAugusti) [inline README.md:302]:
```suggestion
9. Trouvez les variables env appropriées et copiez-les dans le fichier .env. Un ensemble de valeurs par défaut existe dans le fichier `.env.example` à la racine ou vous pouvez utiliser celles du dossier LastPass. Si vous utilisez celles de LastPass et que vous exécutez l'API localement, modifiez `API_HOST_NAME` pour qu'elle pointe vers votre machine locale
```

File: README.md [hunk 11]
```diff
@@ -207,3 +207,214 @@ See the [notification-api](https://github.com/cds-snc/notification-api) README f
...
+
+12. Générer les traductions
+
+`make babel`
+
+13. Installer le npm et construire les actifs 
+
+`npm install` suivi de `npm run build`.
+
+14.  Démarer le service
```
Comment (AntoineAugusti) [inline README.md:320]:
```suggestion
14.  Démarrer le service
```

File: README.md [hunk 12]
```diff
@@ -207,3 +207,214 @@ See the [notification-api](https://github.com/cds-snc/notification-api) README f
...
+
+11. Générer le fichier de version 
+
+`make generate-version-file`
+
+12. Générer les traductions
+
+`make babel`
+
+13. Installer le npm et construire les actifs
```
Comment (AntoineAugusti) [inline README.md:316]:
```suggestion
13. Installer les dépendances npm et construire les actifs 
```

File: README.md [hunk 13]
```diff
@@ -207,3 +207,214 @@ See the [notification-api](https://github.com/cds-snc/notification-api) README f
...
+
+`flask run -p 6012 --host=0.0.0.0``.
+
+15. Pour tester
+
+`pip3 install -r requirements_for_test.txt`
+
+`make test`
+
+## Reconstruire les ficher CSS et JS du frontend
```
Comment (AntoineAugusti) [inline README.md:330]:
```suggestion
## Reconstruire les fichiers CSS et JS du frontend
```

File: README.md [hunk 14]
```diff
@@ -207,3 +207,214 @@ See the [notification-api](https://github.com/cds-snc/notification-api) README f
...
+
+Si vous souhaitez que les fichier JS et CSS soient recompilés en fonction des changements, laissez rouler cette fonction dans un terminal séparé de l'application
```
Comment (AntoineAugusti) [inline README.md:332]:
```suggestion
Si vous souhaitez que les fichier JS et CSS soient recompilés en fonction des changements, laissez tourner cette fonction dans un terminal séparé de l'application
```

File: README.md [hunk 15]
```diff
@@ -207,3 +207,214 @@ See the [notification-api](https://github.com/cds-snc/notification-api) README f
...
+```shell
+    npm run watch
+```
```
Comment (AntoineAugusti) [inline README.md:335]:
```suggestion
```shell
npm run watch
```
```

File: README.md [hunk 16]
```diff
@@ -207,3 +207,214 @@ See the [notification-api](https://github.com/cds-snc/notification-api) README f
...
+
+
+## Mise à jour des dépendances des applications
+
+Le fichier `requirements.txt` est généré à partir du fichier `requirements-app.txt` afin d'épingler des versions de toutes les dépendances imbriquées. Si `requirements-app.txt` a été modifié (ou nous voulons mettre à jour les dépendances imbriquées non épinglées) requirements.txt` devrait être régénérée avec
```
Comment (AntoineAugusti) [inline README.md:340]:
```suggestion
Le fichier `requirements.txt` est généré à partir du fichier `requirements-app.txt` afin d'épingler des versions de toutes les dépendances imbriquées. Si `requirements-app.txt` a été modifié (ou nous voulons mettre à jour les dépendances imbriquées non épinglées) `requirements.txt` devrait être régénérée avec
```

File: README.md [hunk 17]
```diff
@@ -207,3 +207,214 @@ See the [notification-api](https://github.com/cds-snc/notification-api) README f
...
+
+```
+make freeze-requirements
+```
+
+Le fichier `requirements.txt` doit être commiter en même temps que les modifications du fichier `requirements-app.txt`.
```
Comment (AntoineAugusti) [inline README.md:346]:
```suggestion
Le fichier `requirements.txt` doit être ajouté en même temps que les modifications du fichier `requirements-app.txt`.
```

File: requirements-app.txt (hunk 2 above)
Comment (sboisvert) [reply requirements-app.txt:39]: when using pipenv on Ubuntu I was getting some failures. I found https://github.com/pypa/pipenv/issues/2057#issuecomment-384594549 and it solved the problem.

File: README.md [hunk 18]
```diff
@@ -207,3 +207,214 @@ See the [notification-api](https://github.com/cds-snc/notification-api) README f
...
+
+
+## Travailler avec des fichier statiques
+
+Lorsque utilier locallement, les fichier statiques sont desservis par Flask à http://localhost:6012/static/...
```
Comment (AntoineAugusti) [inline README.md:351]:
```suggestion
Lorsque utilisé localement, les fichiers statiques sont servis par Flask à http://localhost:6012/static/...
```

File: README.md [hunk 19]
```diff
@@ -207,3 +207,214 @@ See the [notification-api](https://github.com/cds-snc/notification-api) README f
...
+Lorsque en production ou sur staging, c'est un peu plus compliqué:
+
+![notify-static-after](https://user-images.githubusercontent.com/355079/50343595-6ea5de80-051f-11e9-85cf-2c20eb3cdefa.png)
+
+
+## Traductions
+
+- Le texte dans le code est en anglais
+- Enveloppez votre texte avec `{{ }}`
+- Les traductions sont dans app/translations/csv/fr.csv
```
Comment (AntoineAugusti) [inline README.md:362]:
```suggestion
- Les traductions sont dans `app/translations/csv/fr.csv`
```

File: README.md [hunk 20]
```diff
@@ -207,3 +207,214 @@ See the [notification-api](https://github.com/cds-snc/notification-api) README f
...
+- Pour des conseils sur les formulaires 
+
+Crée une variable
+
+```
+ <div class="extra-tracking">
+  {% set hint_txt = _('We’ll send you a security code by text message') %}
+  {{textbox(form.mobile_number, width='3-4', hint=hint_txt) }}
+ </div>
+```
```
Comment (AntoineAugusti) [inline README.md:377]:
```suggestion
```
<div class="extra-tracking">
  {% set hint_txt = _('We’ll send you a security code by text message') %}
  {{ textbox(form.mobile_number, width='3-4', hint=hint_txt) }}
</div>
```
```

File: README.md [hunk 21]
```diff
@@ -207,3 +207,214 @@ See the [notification-api](https://github.com/cds-snc/notification-api) README f
...
+
+```
+// dans vos fichier JS
+let now_txt = window.polyglot.t("now") ;
+```
+
+- Extrait
+
+Actuellement, il s'agit d'une étape manuelle. Ajoutez une ligne à en.csv et fr.csv dans app/translations/csv/ pour chaque nouvelle  de charactère que vous avez enveloppée. Le format est le suivant : "Texte Anglais", "traduction". Assurez-vous que la chaîne enveloppée que vous ajoutez est unique.
```
Comment (AntoineAugusti) [inline README.md:406]:
```suggestion
Actuellement, il s'agit d'une étape manuelle. Ajoutez une ligne à `en.csv` et `fr.csv` dans `app/translations/csv/` pour chaque nouvelle  de charactère que vous avez enveloppée. Le format est le suivant : `"Texte Anglais", "traduction"`. Assurez-vous que la chaîne enveloppée que vous ajoutez est unique.
```

File: README.md [hunk 22]
```diff
@@ -207,3 +207,214 @@ See the [notification-api](https://github.com/cds-snc/notification-api) README f
...
+
+- Compiler 
+
+```Bash
+make babel
+```
+
+## Utiliser Jinja localement pour tester les changements de modèles
+
+Voir le [notification-api](https://github.com/cds-snc/notification-api) README pour des instructions détaillées.
```
Comment (AntoineAugusti) [inline README.md:416]:
```suggestion
Voir le dépôt [notification-api](https://github.com/cds-snc/notification-api) README pour des instructions détaillées.
```

File: README.md [hunk 23]
```diff
@@ -207,3 +207,214 @@ See the [notification-api](https://github.com/cds-snc/notification-api) README f
...
+
+Fichiers de modèles utilisés dans cette repo : `sms_preview_template.jinja2, email_preview_template.jinja2`
+
+Note : Les tests peuvent échouer si `USE_LOCAL_JINJA_TEMPLATES` est réglé sur `True` dans votre .env
```
Comment (AntoineAugusti) [inline README.md:420]:
```suggestion
Note : les tests peuvent échouer si `USE_LOCAL_JINJA_TEMPLATES` est réglé sur `True` dans votre `.env`
```

File: README.md [hunk 24]
```diff
@@ -207,3 +207,214 @@ See the [notification-api](https://github.com/cds-snc/notification-api) README f
...
+
+```Bash
+make babel
+```
+
+## Utiliser Jinja localement pour tester les changements de modèles
+
+Voir le [notification-api](https://github.com/cds-snc/notification-api) README pour des instructions détaillées.
+
+Fichiers de modèles utilisés dans cette repo : `sms_preview_template.jinja2, email_preview_template.jinja2`
```
Comment (AntoineAugusti) [inline README.md:418]:
```suggestion
Fichiers de modèles utilisés dans ce dépôt : `sms_preview_template.jinja2, email_preview_template.jinja2`
```

**Review by AntoineAugusti — COMMENTED** [review]
Focused on suggesting changes to the README in French. Thanks for your work!

File: README.md (hunk 20 above)
Comment (sboisvert) [reply README.md:375]: celui semble pas avoir fonctioner, but je pense que c'était pour enlever une espace qui aurais pas du etre la.

**Comment by sboisvert** [comment]
> 
> 
> I think we can make the .env variable paragraph a little more clear, and there is currently a merge conflict. Otherwise, looks great, and super happy to see the french translation! Thank you!

How does it look now?

**Review by brdunfield — APPROVED** [review]
👍 Much better!

**Comment by AntoineAugusti** [comment]
@sboisvert You can accept multiple changes at the same time by using the "Add suggestion to batch" next time!

The UI is a bit complicated for proposing changes / accepting them. Here is the help article: https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes

I was so glad when I discovered how to do it 😂

**Pull request merged**